doc/rst/developer/bestPractices/ContributorInfo.rst

```diff
@@ -649,40 +650,28 @@ responsibility is on the developer.
 Reviewer responsibilities
 +++++++++++++++++++++++++
 
+* Test the PR you are reviewing.  Follow the instructions next to the
+  "Merge pull request" button in the "command-line instructions" link
+  to checkout a local copy of the PR for testing.  They will look something
+  like the following:
+
+.. code-block:: bash
+
+  git checkout -b contribusername-prbranchname main
+  git pull https://github.com/contribusername/chapel.git prbranchname
+  // don't forget to recompile before you test
+
+
 * If you're reviewing a commit from a developer outside the Chapel core
   team, be sure their commits are signed via the DCO bot (one of several
   github action checks that will run on each PR).  If they're not, help
   the developer understand the requirement.
 
-  Note that using GitHub's "squash and merge" feature will effectively
-  drop all DCO signature lines from the pull request, and so should
-  not be used on external commits (or potentially even for commits
-  from the core team?) in order to preserve the signed nature of the
-  commits.
-
-  Care may need to be taken when committing third-party code that
-  originates from a different git[hub] repository.  As an example, in
-  one case in the past we brought in a copy of an outside commit that
-  had originally been made in the git repository belonging to one of our
-  third-party packages.  We did that by using git-am to commit a copy of
-  their raw commit (in git-send-mail format) to the appropriate
-  third-party directory in the Chapel repository.  For the commit in our
-  repo, their developer was listed as the author, but the Chapel core
-  team member who did the Chapel commit was listed as the contributor.
-  Had we instead split the original commit apart into its constituent
-  meta-information and patch parts and committed just the patch using
-  git-apply, the Chapel core team member would have been listed as both
-  author and contributor.  In the end it didn't matter because although
-  the outside developer couldn't sign our contributor's agreement, their
-  IP attorneys decided that given their license (which was BSD), their
-  commit constituted publishing the work rather than contributing it,
-  and what Chapel did with it afterward was not their concern.  Also, we
-  would have picked up the same commit the next time we updated our
-  third-party release of that package.  Nevertheless, this gives an
-  example of how tricky this kind of situation can be, and shows why
-  decisions may need to be made (or at least understood) at a high
-  level.
-
+* Care may need to be taken when committing third-party code that
+  originates from a different git[hub] repository.  There is a 
+  contributor's agreement that might be needed.  Have a discussion
```

Comment:
I might add "We also may need to do a licensing review."  Just before "Have a discussion..."